pkg/kubelet/kubelet_pods.go

```diff
@@ -487,6 +499,26 @@ func managedHostsFileContent(hostIPs []string, hostName, hostDomainName string,
 	return buffer.Bytes()
 }
 
+// isIPv6Supported determines whether the host supports ipv6 by check the global ipv6 support sysctl.
+// We use it to determine whether the pod should support ipv6.
+// But there are going to be edge cases we don't get right,
+// e.g. the user may have IPv6 enabled in the host netns,
+// but for some reason disable IPv6 in the pod netns.
+// But because kubelet cannot actually know about the loopback interface of the pod,
+// we cannot do something completely correct,
+// so we can only do simple and basically correct things.
+func isIPv6Supported() (bool, error) {
```

Comment:
@thockin @aojea I guess that's what we expect?

Reply:
Is there a need for a `kubelet_pods_windows.go` to hold the windows implementation version?